src/storage/in_mem_storage_structure/in_mem_column.cpp

```diff
@@ -14,12 +14,11 @@ InMemColumn::InMemColumn(std::string filePath, LogicalType dataType, bool requir
     // TODO(Guodong): Separate this as a function.
     switch (this->dataType.getLogicalTypeID()) {
     case LogicalTypeID::STRUCT: {
-        auto childTypes = common::StructType::getStructFieldTypes(&this->dataType);
-        auto childNames = common::StructType::getStructFieldNames(&this->dataType);
-        childColumns.resize(childTypes.size());
-        for (auto i = 0u; i < childTypes.size(); i++) {
+        auto fieldTypes = common::StructType::getFieldTypes(&this->dataType);
+        childColumns.resize(fieldTypes.size());
```

Comment:
I think it is better to use reserve, and call push_back in the for loop(which we did in other loops).